docs/file-events.yml

```diff
@@ -788,10 +788,11 @@ components:
             $ref: '#/components/schemas/FileEventV1'
         nextPgToken:
           type: string
+          nullable: true
           description: Use as the pgToken value in another request to indicate the
             starting point for additional page results. nextPgToken is null if there
             are no more results or if pgToken was not supplied.
-          example: 0_147e9445-2f30-4a91-8b2a-9455332e880a_973435567569502913_986467523038446097_163
+          example: null
```

Comment:
page token needs to be set to null (and the property needs to be specified as nullable) so that the pages don't link infinitely when returned